rest-api/2-token-validation/Startup.cs

```diff
@@ -32,11 +35,26 @@ public void ConfigureServices(IServiceCollection services)
                     options.ClientId = "resource1";
                     options.ClientSecret = "secret";
 
-                })
+                });
```

Comment:
We should also clear inbound claims here, to avoid the annoying mapping that the default JWT handler does in ASP.NET.

Reply:
Fixed, adde to 9-Complete startup.cs